flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/expressions/PlannerExpressionConverter.scala

```diff
@@ -94,6 +94,12 @@ class PlannerExpressionConverter private extends ApiExpressionVisitor[PlannerExp
 
           case AS =>
             assert(args.size >= 2)
+            args.drop(1).foreach {
```

Comment:
Why do we validate it here? The whole `PlannerExpressionConverter` is already past validationPhase.

Reply:
Ok, I validate it because of the test, i.e., `testInvalidRenameColumns()` in `ColumnsOperationValidationTest`. I think you make a good point here, maybe we need to validate the as in Column Expander Rule?